docker/rubocop

```diff
@@ -19,6 +19,6 @@ if [[ -z "${changed_ruby_files// }" ]]; then
 else
   docker run -v $DIR/..:/code \
              -w /code \
-             openstax/kitchen.ci:latest \
+             openstax/cookbook.ci:latest \
```

Comment:
I have made this fix related to failing check, but I wasn't sure about it because in @kswanson33 [PR](https://github.com/openstax/cookbook/pull/1) this check has passed but maybe is is related to the container rebuilding: 
![image](https://user-images.githubusercontent.com/55203598/152353436-003a7340-dc7c-4c19-8282-40627455f461.png)